app/js/components/maintenance/maintenanceTable/maintenanceTableActions.jade

```diff
@@ -20,10 +20,9 @@
         ng-mousedown="checkboxRowClick($event, action.id)")
         td.text-center(ng-show='edit')
           input(ng-model='checkboxes.items[action.id]', type="checkbox")
-        td.overflow
-          span.icon-inline(ng-if='action._type === 2')
-            i.fa.fa-exclamation-circle(tooltip="{{'This action is already part of another plan' | translate}}")
-          a.action(ng-click='table.showSystemModal(table.params.item, action.rule)') {{action.display}}
+        td.overflow(ng-class='{icon-inline: action._type === 2}')
```

Comment:
icon-inline needs to be in quotes. Chrome browser console is reporting a syntax error